operator/deploy/kiali/kiali_cr.yaml

```diff
@@ -257,6 +257,7 @@ spec:
 # namespace: The Kubernetes namespace that holds the Tracing service (if empty, assumes the same value as deployment.namespace)
 # service: The Kubernetes service name for tracing. Kiali uses this to connect within the cluster to Jaeger.
 # port: The kubernetes port where the tracing service is mount. Default port is Jaeger port, 16686. Autodiscovered when service is not specified.
+# in_cluster_url: Set URL for in-cluster access. Example: "http://jaeger-query.istio-system:16686".
```

Comment:
As default value is empty. Perhaps worth to indicate if when empty what happens (i.e. it defaults to autodiscovery, or service would be disabled, etc.
Perhaps this comment can apply to other fields too.

Reply:
please put all these new values in alphabetical order. We need to keep these in order to help diff future versions and to just make things clean and orderly to easily find things. Move those "in_cluster_url" above "insecure_skip_verify"